composables/useFeature.ts

```diff
@@ -0,0 +1,63 @@
+import type { ApiPoi } from '~/lib/apiPois'
+
+type LabelType = 'details' | 'popup'
+interface Options {
+  type?: LabelType
+}
+
+export default function useFeature(feature: ApiPoi, options: Options = {
+  type: undefined,
+}) {
+  //
+  // Composables
+  //
+  const { locale, fallbackLocale } = useI18n()
+
+  //
+  // Data
+  //
+  const featureName = feature.properties.name || '' // INFO: Here we could report missing name in Sentry for exemple
+  const { type } = options
+
+  //
+  // Computed
+  //
+  const featureSeoTitle = computed(() => {
+    const categoryLabel = getCategoryLabel()
+
+    return categoryLabel ? `${categoryLabel} - ${featureName}` : featureName
+  })
+
+  //
+  // Methods
+  //
+  function getCategoryLabel() {
+    const fallback = fallbackLocale.value.toString()
+
+    if (!feature.properties.editorial)
+      return undefined
+
+    if (type === 'details') {
+      if (feature.properties.editorial.class_label_details && feature.properties.editorial.class_label_details[locale.value])
+        return feature.properties.editorial.class_label_details[locale.value]
+
+      if (feature.properties.editorial.class_label_details && feature.properties.editorial.class_label_details[fallback])
+        return feature.properties.editorial.class_label_details[fallback]
```

Comment:
1. label_details[fr]
2. labe[fr]
3. label_details[en]
4. label[en]

Reply:
yes I haven't commited my final changes, I wanted to clean my commits first